src/main/java/lotto/domain/LottoTickets.java

```diff
@@ -9,8 +9,14 @@
 public class LottoTickets {
     private final List<LottoTicket> lottoTickets;
 
+
     public LottoTickets(int ticketCount, LottoGenerator lottoGenerator) {
-        List<LottoTicket> lottoTickets = new ArrayList<>();
+        this(new ArrayList<>(), ticketCount, lottoGenerator);
+    }
+
+    public LottoTickets(List<LottoTicket> lottoTicketsInput, int ticketCount,
+        LottoGenerator lottoGenerator) {
+        List<LottoTicket> lottoTickets = new ArrayList<>(lottoTicketsInput);
```

Comment:
소소한 성능향상 👍

Reply:
[❓질문]
어떤 점에서 성능이 향상된 건지 알 수 있을까요? 제 의도는 외부와의 연결고리를 끊어주기 위해 방어적 복사를 사용해보려고 새로운 ArrayList를 만든 것인데 어떤 점에서 성능이 향상된 건지 잘 모르겠습니다😅

Reply:
앗 해당 부분은 방어적 복사 코드였는데 기본 Capacity를 넣어준거로 잘못 봤네요..!
혼란을 드려 죄송합니다 😭 


```suggestion
        List<LottoTicket> lottoTickets = new ArrayList<>(lottoTicketsInput.size() + ticketCount);
```

와 같이 작성했을 때 어느 부분에서 성능향상이 있을지에 대한 리뷰였습니다
ArrayList 내부 동작을 공부하는데 도움이 될거에요!

Reply:
```private static final int DEFAULT_CAPACITY = 10; 
```

ArrayList가 처음 배열의 크기는 10으로 되어있고 크기를 재산정해서 늘려주네요! 처음에 크기를 지정해주면 재산정해서 새로 늘려주는 작업이 줄어들어 성능향상이 되는군요! 😀👍